sections/staking/components/helper.ts

```diff
@@ -45,3 +45,11 @@ export function getTransferableAmountFromBurn(
 		)
 	);
 }
+
+export function sanitiseValue(value: BigNumber) {
```

Comment:
I believe we use American English, so I guess `sanitizeValue`?

Reply:
Lol who said that